style.css

```diff
@@ -1219,6 +2777,67 @@ body {
     border-right: 4px solid #000;
     border-top: 4px solid #000;
   }
+
+  .second-page__article {
+    width: 300px;
+  }
+
+  .second-page__title {
+    font-size: 25px;
+  }
+
+  .second-page__subtitle {
+    font-size: 12px;
+  }
+
+  .second-page__description {
+    width: 300px;
+    font: 600 11px/16px 'Montserrat-Regular', sans-serif;
+  }
+
+  .eclipse {
+    width: 28px;
+    height: 28px;
+    margin-right: 14px;
+  }
+
+  .third-page {
+    background: linear-gradient(to right, #EBE3D8 120px, #000 120px, #000 168px, #EBE3D8 168px);
```

Comment:
Codacy found an issue: [Expected "#EBE3D8" to be "#ebe3d8" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=6358146)